src/client/Microsoft.Identity.Client/Instance/Validation/AdfsAuthorityValidator.cs

```diff
@@ -47,6 +48,7 @@ public AdfsAuthorityValidator(RequestContext requestContext)
                         a => a.Rel.Equals(Constants.DefaultRealm, StringComparison.OrdinalIgnoreCase) &&
                              a.Href.Equals(resource)) == null)
                 {
+                    _requestContext.Logger.Error($"Authority validation failed due to invalid authority. Authority: {authorityInfo.CanonicalAuthority}");
```

Comment:
nit:
```suggestion
                    _requestContext.Logger.Error($"Authority validation failed because the configured authority is invalid. Authority: {authorityInfo.CanonicalAuthority}");
```